python/lsst/ts/linearstage/controllers/zaber_lst.py

```diff
@@ -180,14 +387,14 @@ async def enable_motor(self):
         value : `bool`
             True to enable the motor, False to disable the motor.
         """
-        pass
+        self.log.info("Zaber stage has no brake and so this is a noop.")
```

Comment:
If this class is initialized correctly then the logger is coming from the CSC. That means that this line will be logged into EFD. Is that really what you want? Same for line 397.